test/helper.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
```

Comment:
This file contains some helper configuration for tests and MUST be required in all test files that access the database.